SAT/SATSolver.hpp

```diff
@@ -302,11 +276,6 @@ class PrimitiveProofRecordingSATSolver : public SATSolverWithAssumptions {
       _refutation->setInference(_refutationInference);
     }
 
-  virtual ~PrimitiveProofRecordingSATSolver() {
-    // cannot clear the list - some inferences may be keeping its suffices till proof printing phase ...
```

Comment:
OK, because `SATSolver` already has a virtual destructor?

Reply:
Exactly - and this one doesn't do anything, so (I think) it's pointless.